mixins/__tests__/timeUtils.spec.ts

```diff
@@ -0,0 +1,32 @@
+import Vue from 'vue';
+import timeUtils from '~/mixins/timeUtils.ts';
+
+const TestComponent = new Vue({
+  mixins: [timeUtils]
+});
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Why? This line shouldn't be causing TypeScript issues.

Reply:
Hmm it was causing me issues, I'll have another look.

Reply:
I've added @types/jest which should fix any problems